app/helpers/course_details_helper.rb

```diff
@@ -63,8 +63,20 @@ def path_for_course_details(trainee)
     edit_trainee_course_education_phase_path(trainee)
   end
 
+  def language_specialism_options
+    options = language_specialisms.map do |language|
+      [format_language(language).upcase_first, language]
+    end
+
+    [["", ""]] + options
```

Comment:
what's the function of this `[["", ""]]`?

Reply:
This is for the no value option. `include_blank` doesn't seem to work in the 'normal way' here.